R/construct_design.R

```diff
@@ -21,6 +21,8 @@
 #'
 #' @return a list of two functions, the \code{design_function} and the \code{data_function}. The \code{design_function} runs the design once, i.e. draws the data and calculates any estimates and estimands defined in \code{...}, returned separately as two \code{data.frame}'s. The \code{data_function} runs the design once also, but only returns the final data.
 #'
+#' @name declare_design
```

Comment:
Is declare_design come back from the grave?

Reply:
Hah, no just naming the file :)